test/player-test.js

```diff
@@ -2,21 +2,89 @@ import chai from 'chai';
 const expect = chai.expect;
 
 import Player from '../src/scripts/Player.js';
+import Survey from '../src/scripts/Survey.js';
+
 
 describe('Player:', () => {
-  let player;
+  let playerGuess;
+  let survey;
+  let data = {
+      surveys: [{
+        id: 1,
+        question: 'If You Drew Homer Simpson’s Name In A Secret Santa Exchange, What Would You Buy Him?'
+      }, ],
+      answers: [{
+          answer: 'Beer',
+          respondents: 67,
+          surveyId: 1
+        },
+        {
+          answer: 'Bowling Ball',
+          respondents: 5,
+          surveyId: 1
+        },
+        {
+          answer: 'Donuts',
+          respondents: 24,
+          surveyId: 1
+        },
+      ]
+    };
+  let player1;
   let name = 'Caleb';
-
+  let playerNum = 1;
   beforeEach(() => {
-    player = new Player(name);
+    player1 = new Player(name, playerNum);
+  });
+  it('should be a function', function() {
+    expect(Player).to.be.a('function');
   });
-    it('should be a function', function() {
-      expect(Player).to.be.a('function');
-    });
 
-    describe('Default Values:', () => {
+  describe('Default Values:', () => {
+    it('should start with 0 total points', function() {
+      expect(player1.totalPoints).to.equal(0);
+    });
     it('should store the name of the player', function() {
-      expect(player.name).to.equal('Caleb');
+      expect(player1.name).to.equal('Caleb');
+    });
+    it('should have a playerNum', function() {
+      expect(player1.playerNum).to.equal(1);
+    });
+    it('should start with an empty array of incorrectGuesses', function() {
+      expect(player1.incorrectGuesses).to.deep.equal([]);
+    });
+    it('should start with a 0 multiplier', function() {
+      expect(player1.multiplier).to.equal(0);
     });
+    describe('Method Values:', () => {
```

Comment:
I'd recommend still grouping your tests for each method by the methods themselves, rather than just putting all methods together. Still, some organization is better than none! Just may keep things a little more cleanly divided